servicetalk-opentelemetry-http/src/main/java/io/servicetalk/opentelemetry/http/ServiceTalkHttpAttributesGetter.java

```diff
@@ -33,41 +33,38 @@
 import static java.util.Collections.singletonList;
 import static java.util.Collections.unmodifiableList;
 
-final class ServiceTalkHttpAttributesGetter implements
-            HttpClientAttributesGetter<HttpRequestMetaData, HttpResponseMetaData>,
-            HttpServerAttributesGetter<HttpRequestMetaData, HttpResponseMetaData> {
+abstract class ServiceTalkHttpAttributesGetter {
 
-    static final ServiceTalkHttpAttributesGetter INSTANCE = new ServiceTalkHttpAttributesGetter();
+    static final HttpClientAttributesGetter<HttpRequestMetaData, HttpResponseMetaData> CLIENT_INSTANCE =
+            new ClientGetter();
+
+    static final HttpServerAttributesGetter<HttpRequestMetaData, HttpResponseMetaData> SERVER_INSTANCE =
+            new ServerGetter();
```

Comment:
Note that in both of the getter types, we need these two different sub-classes because the ClientGetter and ServerGetter have some new default methods the the same signature and the java compiler doesn't know which to use.